docs/pages/guides/apple-privacy.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Privacy Manifests
+description: Learn about configuring iOS privacy manifests for your mobile app.
+---
+
+import { ConfigReactNative } from '~/components/plugins/ConfigSection';
+
+If you're using a native iOS library that uses a "restricted reason" APIs, you'll need to configure an iOS privacy manifest to declare why you're including native code calling those APIs.
+
+More details and a list of "required reason" APIs can be found in the [Apple Developer Documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).
+
+> **info** The information and steps included in this document are still in development and may change, either due to new tools built for this purpose or new requirements from Apple.
+
+## What is a Privacy Manifest?
+
+A privacy manifest is a file named **PrivacyInfo.xcprivacy** that is included in your iOS native project. This file is used to declare why the app includes native code that calls into certain APIs that Apple considers sensitive.
+
+Those APIs currently include accessing UserDefaults, file timestamp APIs, system boot time APIs, disk space APIs, and active keyboard APIs. Apple considers it an open list that can be expanded in the future.
```

Comment:
```suggestion
These APIs currently include accessing UserDefaults, file timestamp, system boot time, disk space, and active keyboard. Apple considers it an open list that can be expanded in the future.
```

- Let's remove APIs after each API name, it's redundant since we mention that they are APIs as the first thing in the sentence.